vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -31,230 +31,6 @@
 
 namespace Microsoft::ReactNative {
 
-// ViewShadowNode
```

Comment:
move the class declaration to the header, but keep the methods' implementation in the cpp file